libr/core/cmd_type.c

```diff
@@ -745,6 +746,40 @@ static int cmd_type(void *data, const char *input) {
 				r_core_cmd_help (core, help_msg_ts);
 			}
 			break;
+		case 'c': { //tsc
+			char *name = NULL;
+			char *varname = NULL;
+           		SdbKv *kv;
+           		SdbListIter *iter;
+			int i, j;
+        		SdbList *l = sdb_foreach_list (TDB, true);
+        		ls_foreach (l, iter, kv) {
+        			if (!strcmp (sdbkv_value (kv), "struct")) {
```

Comment:
For me it looks like a too deep nesting. What about extracting a part in a separate function instead?

Reply:
Which part do you mean?

Reply:
remove the ! and continue; this way you wont be nesting that much